### PR TITLE
test: add test for per registry credential helpers

### PR DESCRIPTION
**Pull request by thesayyn**

### CODE REVIEW

File: .bazelrc [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 build --incompatible_strict_action_env
 build --nolegacy_external_runfiles
+build --symlink_prefix=/
```
Comment (alexeagle) [inline .bazelrc:7]: why? it seems like it belongs in your personal bazelrc

**Review by alexeagle — APPROVED** [review]

File: .bazelrc (hunk 1 above)
Comment (thesayyn) [reply .bazelrc:7]: `bats -r .` recurses into bazel-bin causing infinite loops.  this belongs in e2e/auth actually. will fix it

Comment (thesayyn) [reply .bazelrc:7]: oh, it's a symlink. can't do that. I'll remove this and put into .bazelrc.user

Comment (thesayyn) [reply .bazelrc]: done.

**Pull request merged**